content/docs/aws/features.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Kubeflow on AWS Features"
+weight = 200
++++
+
+## Reasons to use Kubeflow on AWS
+
+Running Kubeflow on Amazon EKS brings the following optional and configurable features:
+
+* You can manage your Amazon EKS cluster provisioning with **eksctl** and easily choose between multiple compute and GPU worker node configurations.
```

Comment:
Suggestion: Add a link for each of the features on this page, so people who don't know what they are can easily find more info.